test/local/random.js

```diff
@@ -0,0 +1,21 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict'
+
+const assert = require('insist')
+
+const base32 = require('../../lib/db/random')
+
+describe('random', () => {
+  it('should generate random code', () => {
+    return base32(10)
+      .then(code => {
+        assert.equal(code.length, 10)
+        assert.equal(code.indexOf('I'), -1, 'should not contain I')
```

Comment:
@deeptibaghel Whoops, just noticed these are upper case. We should change to check for lower case.